### ISSUE TITLE
Start a custom jar

### ISSUE THREAD

**Issue by Xanderplayz16** [mindstorm38/portablemc#212]
I have a custom jar to launch, and the API docs have nothing on like specifying a path to the jar, or anything like that.

**Comment by mindstorm38** [comment]
Hi! To launch a custom JAR you have to conform to how `.minecraft` directories work (so-called "main dir"). In fact, you need to create a directory for your version under `<main dir>/versions`, and inside it, you also need to add a version metadata file `<your version>.json`, this metadata is really important as it tells the launcher how to build the command line to start your jar, which libraries to use, etc. You can check how mojang's versions are crafted and base your one on this.

Here is a sample directory structure, if your main dir is `.minecraft` and your custom version is named `hello`:
- `.minecraft`
  - `versions`
    - `hello`
      - `hello.json`
      - `hello.jar`

**Comment by Fugach** [comment]
I see you reacted to my message, can you write me a guide how to do it, please?
@mindstorm38

**Comment by mindstorm38** [comment]
Yes this is very not obvious, and I debatted with myself a long time to just relax this rule. What do you think? I'm more and more in favor of relaxing the placement of "common" options.

@Fugach @Ristovski

**Comment by theorzr** [comment]
Just release v5.0.3 addressing the problem with common arguments!
https://github.com/theorzr/portablemc/releases/tag/v5.0.3

Thanks again @Ristovski and @Fugach for the suggestion!
